Exiled.API/Features/Player.cs

```diff
@@ -1146,7 +1146,12 @@ public void ResetInventory(List<ItemType> newItems)
         /// <summary>
         /// Clears the player's inventory.
         /// </summary>
-        public void ClearInventory() => Inventory.items.Clear();
+        public void ClearInventory() => Inventory.Clear();
```

Comment:
`Inventory::Clear()` also affects the dropping of `AmmoBox`. I think you shouldn't have changed it.